### PR TITLE
[AIG]Initial websocket documentation

### PR DESCRIPTION
**Pull request by daisyfaithauma**
### Summary

Initial websocket documentation

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by G4brym — APPROVED** [review]

File: src/content/docs/ai-gateway/configuration/websockets-api.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+WebSockets are long-lived TCP connections that enable bi-directional, real-time communication between client and server. Unlike HTTP connections, which require repeated handshakes for each request, WebSockets maintain the connection, supporting continuous data exchange with reduced overhead. WebSockets are ideal for applications needing low-latency, real-time data, such as voice assistants.
+
+## Key benefits
+
+- **Reduced Overhead**: Avoid overhead of repeated handshakes and TLS negotiations by maintaining a single, persistent connection.
+- **Provider Compatibility**: Works with all AI providers in AI Gateway. Even if your chosen provider does not support WebSockets, we handle it for you, managing the requests to your preferred AI provider.
+
+## Setting up
+
+1. Generate an AI Gateway token with appropriate AI Gateway Run and opt in to using an authenticated gateway.
```
Comment (kathayl) [inline src/content/docs/ai-gateway/configuration/websockets-api.mdx:22]: is this the correct way to show RUN permission?

File: src/content/docs/ai-gateway/configuration/websockets-api.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+2. Modify your Universal Endpoint URL by replacing `https://` with `wss://` to initiate a WebSocket connection:
+   ```
+   wss://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_id}
+   ```
+3. Open a WebSocket connection authenticated with a Cloudflare token with the AI Gateway Run permission.
```
Comment (kathayl) [inline src/content/docs/ai-gateway/configuration/websockets-api.mdx:27]: is this the correct way to show RUN permission?

File: src/content/docs/ai-gateway/providers/universal.mdx [hunk 3]
```diff
@@ -30,3 +30,43 @@ You can use the Universal endpoint to contact every provider. The payload is exp
 <Render file="universal-gateway-example" />
 
 The above will send a request to Workers AI Inference API, if it fails it will proceed to OpenAI. You can add as many fallbacks as you need, just by adding another JSON in the array.
+
+## Websockets API (beta)
```
Comment (kathayl) [inline src/content/docs/ai-gateway/providers/universal.mdx]: should beta be in words or be the beta label

File: src/content/docs/ai-gateway/configuration/websockets-api.mdx (hunk 1 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/configuration/websockets-api.mdx:22]: I got it right off the documentation.

File: src/content/docs/ai-gateway/providers/universal.mdx (hunk 3 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/providers/universal.mdx]: Changed

File: src/content/docs/ai-gateway/configuration/websockets-api.mdx (hunk 1 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/configuration/websockets-api.mdx:22]: I got it right off the documentation.

File: src/content/docs/ai-gateway/configuration/websockets-api.mdx (hunk 2 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/configuration/websockets-api.mdx:27]: I got it right off the documentation.

File: src/content/docs/ai-gateway/configuration/websockets-api.mdx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+## When to use webSockets?
+
+WebSockets are long-lived TCP connections that enable bi-directional, real-time communication between client and server. Unlike HTTP connections, which require repeated handshakes for each request, WebSockets maintain the connection, supporting continuous data exchange with reduced overhead. WebSockets are ideal for applications needing low-latency, real-time data, such as voice assistants.
+
+## Key benefits
+
+- **Reduced Overhead**: Avoid overhead of repeated handshakes and TLS negotiations by maintaining a single, persistent connection.
+- **Provider Compatibility**: Works with all AI providers in AI Gateway. Even if your chosen provider does not support WebSockets, we handle it for you, managing the requests to your preferred AI provider.
+
+## Setting up
```
Comment (Maddy-Cloudflare) [inline src/content/docs/ai-gateway/configuration/websockets-api.mdx]:
```suggestion
## Set up WebSockets API
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**